utils/dyn-config.c

```diff
@@ -465,12 +475,53 @@ glusterBlockLoadConfig(gbConfig *cfg, bool reloading)
     return -1;
   }
 
-  glusterBlockParseOptions(cfg, buf, len, reloading);
+  glusterBlockParseOptions(cfg, buf, len, getLogDir);
 
   GB_FREE(buf);
   return 0;
 }
 
+
+char *
+glusterBlockDynConfigGetLogDir(void)
+{
+  gbConfig *cfg = NULL;
+  char *logDir = NULL;
+  char *configPath = GB_DEF_CONFIGPATH;
+  int ret;
+
+
+  if (GB_ALLOC(cfg) < 0) {
+    MSG(stderr, "Alloc GB config failed for configPath: %s!\n", configPath);
```

Comment:
With the recent fixes '\n' is no more needed here.
Please fix this else where too.

Reply:
Will fix it.
Thanks.